.env.template

```diff
@@ -63,3 +63,11 @@ export NOTIFICATION_FIREBASE_CREDENTIALS='{"format":"minified JSON";"see":"https
 # and can be left blank
 export NEW_RELIC_APPLICATION_NAME=Backend-dev
 export NEW_RELIC_LICENSE_KEY=
+
+# FusionAuth credentials
+# See https://fusionauth.io/docs/v1/tech/apis/api-keys
+export FUSION_AUTH_HOST=${FUSION_AUTH_HOST}
+export FUSION_AUTH_KEY=${FUSION_AUTH_KEY}
+
+# The location of the permanent server (e.g. "dev.permanent.org")
+export SERVER_NAME=${SERVER_NAME}
```

Comment:
Is `SERVER_NAME` the best thing to call this? (`SERVER_DOMAIN` comes to mind as another candidate)

Reply:
Thank you!  Fixed.